app/controllers/bots_controller.rb

```diff
@@ -7,6 +7,9 @@ def index
   def show
     @user = User.find(params[:user_id])
     @bot = Bot.find(params[:id])
+    @sentences = @bot.interactions.where(category: 'Sentence').order("Sentence")
+    @keywords = @bot.interactions.where(category: 'Keyword').order("Keyword1")
+    @combo = @bot.interactions.where(category: 'Combo').order("Keyword1")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.